components/dashboard/src/repositories/detail/general/RemoveConfigurationModal.tsx

```diff
@@ -39,7 +34,7 @@ export const RemoveConfigurationModal: FunctionComponent<RemoveProjectModalProps
                 name: configuration.name ?? "",
                 description: configuration.cloneUrl ?? "",
             }}
-            buttonText="Remove Configuration"
+            buttonText="Remove Repository"
```

Comment:
Isn't it be a confusing UX for having `Remove Repository` & `Remove Configuration`?

I'd suggest to stick with `Remove Configuration`

![image](https://github.com/gitpod-io/gitpod/assets/55068936/aab8636a-8c9f-4164-89a8-da43b1c87b0c)